src/spyglass/common/common_dandi.py

```diff
@@ -0,0 +1,168 @@
+import os
+import dandi.organize
+import dandi.validate
+from dandi.validate_types import Severity
+import fsspec
+import pynwb
+import h5py
+from uuid import uuid4
+from fsspec.implementations.cached import CachingFileSystem
+
+import datajoint as dj
+
+from spyglass.common.common_usage import Export
+from spyglass.settings import export_dir
+
+from dandi.consts import known_instances
+
+dev_instance = known_instances["dandi-staging"]
+
+schema = dj.schema("common_dandi")
+
+
+@schema
+class DandiPath(dj.Manual):
+    definition = """
+    -> Export.File
+    ---
+    dandiset_id: str
```

Comment:
Is the `dandiset_id` the same for all files in a given `Export`? If so, I might avoid some redundancy by having `DandiPath` be a master table with one part per `Export.File`

Reply:
As written, the dandiset ID is common for everything in export.  However I could see a future case where an analysis file is used in multiple papers and rather than re-uploading a new copy of the file to dandi you would point to it's existing version in a different dandiset.  I might lean towards leaving the flexibility for this in the table structure